### DISCUSSION TITLE
YTVanced legal issue

### DISCUSSION THREAD

**Discussion by skwowet** [TeamVanced/VancedManager#718] [General]
Hi there,

I have been using YTVanced now for a quite few days, a great app with awesome features 🎉
Was wondering about the legal issues, since Youtube revenue is based on ads and YTVanced remove those out of the box. Don't they sue YTVanced for this? or is just okay to Youtube services for non commercial use?

it would be great if someone could explain the stuff behind this!

Thanks in-advance!

**Comment by KevinX8** [comment]
https://adblockplus.org/blog/five-and-oh-look-another-lawsuit-upholds-users-rights-online

It's completely legal to block ads, not sure where the misconception comes from

**Comment by skwowet** [comment]
@KevinX8 that's great, but my question is we are using Youtube service itself right!? won't it be against their policy?
One such, incident recently i came across was, suing of Groovy & Rhythm (discord based bots) for playing music from the Youtube using url.

Ps: i'm not sure about the Yt policy tho, since Groovy was a commercial bot with premium features idk what was exactly against their policy.

**Reply by KevinX8** [reply]
Those music bots actually count as piracy tools as they bypass DRM and also are commercial products yes, Vanced *is* against YouTubes policy, but so are ad blockers, you use them at your own risk but regarding the law it's completely legal
